modules/creating-a-machine-pool-cli.adoc

```diff
@@ -45,7 +45,7 @@ endif::openshift-rosa[]
 ifdef::openshift-rosa[]
 <8> Optional: Specifies the worker node disk size. The value can be in GB, GiB, TB, or TiB. Replace `<disk_size>` with a numeric value and unit, for example `--disk-size=200GiB`.
 <9> Optional: For Multi-AZ clusters, you can create a machine pool in a Single-AZ of your choice. Replace `<az>` with a Single-AZ.
-<10> Optional: For machine pools in clusters that do not have Red Hat managed VPCs, you can select additional custom security groups to use in your machine pools. You must have already created the security groups and associated them with the VPC you selected for this cluster. For more information, see the requirements for _Security groups_ in _Prepare your environment_.
+<10> Optional: For machine pools in clusters that do not have Red Hat managed VPCs, you can select additional custom security groups to use in your machine pools. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for _Security groups_ in _Prepare your environment_.
```

Comment:
I cannot see a section "Security groups" in the "Prepare your environment" from the preview pages, is that correct?

Reply:
It's here, but I appreciate that this is difficult to find from this: https://68185--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs#rosa-security-groups_rosa-sts-aws-prereqs

To be totally accurate it would be "see the requirements for _Security groups_ in _Prepare your environment_ > _Detailed requirements for deploying ROSA using STS_ > _Provisioned AWS infrastructure_. OpenShift docs style tends to prevent us from being allowed to use direct links inside a module, but IMO this should be an exception.

Reply:
Ah, thanks. I could not find it earlier.

Reply:
Updated the above (and other similar references) to refer to Additional resources so it should be easier for users to find in the built docs.